distributed/worker.py

```diff
@@ -1156,10 +1156,24 @@ def send_to_worker(self, address, msg):
         ):
             max_connections = max_connections * 2
 
+        if self.paused:
+            max_connections = 1
+            throttle_msg = " Throttling outgoing connections because worker is paused."
+        else:
+            throttle_msg = ""
+
         if (
             max_connections is not False
-            and self.outgoing_current_count > max_connections
+            and self.outgoing_current_count >= max_connections
```

Comment:
I think this should be a `>=`.

We don't increment `outgoing_current_count` until a few lines down (L1179 in my version).

Suppose, we have `max_connections=5` and were already at that (`self.outgoing_current_count=5`), and we have a new request. To service that, we'd need a 6th connection, so we want to reply busy (i.e. `>=`).

@mrocklin can you double check my logic here?

Reply:
Sure.  That makes sense to me